data_steward/dev_requirements.txt

```diff
@@ -1,10 +1,12 @@
 coverage==4.5.2
 pylint==2.4.4
 modernize==0.7
+mo-future==2.48.19205
 unittest-xml-reporting>=2.5.2,==2.*
 yapf==0.29.0
 lxml==4.5.0
 beautifulsoup4==4.8.2
 soupsieve==1.9.5
 bs4==0.0.1
 pipdeptree==1.0.0
+wheel
```

Comment:
I assume these changes were needed to support 3.6; could you provide more details as to why?

Reply:
`mo-future` was referenced in the notes [here](https://docs.google.com/document/d/1wb8PRcqjDsPV_Q0WssCsSQpmazCVk-mzqgiI4DyP0c0/edit#heading=h.ooz2v8wi2mun).  Since we aren't using the slimmed down requirements files, I am trying to run the integration tests without these two additions.  The unit tests ran fine.  I'll wait on the integration tests to finish before removing from the PR.